### ISSUE TITLE
Icon aspect working for height, but not width

### ISSUE THREAD

**Issue by viharm** [notclickable-jordan/starbase-80#28]
Hi, I updated from 1.1.6 to 1.5.1 today, and noticed a few issues. This is the first.

It appears that issue #8 is partly back. If `iconAspect` is set to `width`, it somehow tries to fit it into a square. But `height` works by making the icon bigger, and making the text part of the tile smaller.

Not sure if this has anything to do with the rewrite.

**Comment by notclickable-jordan** [comment]
Can you post a config.json fragment and a screenshot of what you're seeing?

**Comment by viharm** [comment]
> Can you post a config.json fragment and a screenshot of what you're seeing?

Config:
```json
{
        "name": "Time & Date",
        "uri": "https://www.timeanddate.com",
        "description": "Time & date tools",
        "icon": "/Custom/TimeAndDate_Logo.png",
        "iconColor": "#FFF",
        "iconBG": "transparent",
        "iconAspect": "width",
        "iconBubble": false
      }
```

Screenshot (arrows are for annotation)

![image](https://github.com/notclickable-jordan/starbase-80/assets/2663937/e1a55263-e7a4-4b75-845e-d8b5a8e2dc1e)

**Comment by notclickable-jordan** [comment]
Fixed in v1.5.2

**Comment by viharm** [comment]
The fix works. Thanks.
